src/App.js

```diff
@@ -6,33 +6,52 @@ import { fetchBusinesses } from './services/yelp';
 function App() {
   const [businesses, setBusinesses] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [zip, setZip] = useState('97034');
+  const [search, setSearch] = useState('');
 
   // TODO -- add state for zip / search and add event listeners to the inputs
 
   useEffect(() => {
     const fetchData = async () => {
       const data = await fetchBusinesses();
-      setBusinesses(data);
+      setBusinesses(data.businesses);
       setLoading(false);
     };
     fetchData();
   }, []);
 
   // TODO -- add event for button click to handle calling fetchBusinesses with zip / search
 
+  const handleClick = async () => {
+    const data = await fetchBusinesses(zip, search);
+    setLoading(false);
+    return setBusinesses(data.businesses);
```

Comment:
The app appears to throw an error when a search parameter is set without a zip code. We can handle this a few ways, perhaps by setting an error message or setting a default zip code if no zip code is entered.